csrc/ops/alias.cpp

```diff
@@ -244,8 +244,6 @@ TensorView* squeeze(TensorView* x, const std::vector<int64_t>& dims) {
           out_domain.push_back(id->cloneWithoutRFactor());
           continue;
         }
-        NVF_CHECK(
```

Comment:
Please leave a comment about the deviation of behavior

Reply:
Noted in the docstrings for both signatures of `squeeze` in `alias.h`.